templates/base/server/services/auth/dependencies.py

```
@@ -0,0 +1,69 @@
+
+
+async def _get_current_user(
+    credentials=Depends(auth_scheme),
+    session: AsyncSession = Depends(get_session),
+    cache: aioredis.Redis = Depends(get_cache),
+    token_manager: TokenManager = Depends(get_token_manager),
+):
+    try:
+        verified_token = token_manager.verify_access_token(credentials.credentials)
+    except JWTError:
+        raise InvalidCredentialsException()
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)